plugins/catalog-backend/examples/locationref.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: backstage.io/v1beta1
+kind: LocationRef
+metadata:
+  name: location-1
+spec:
+  type: github
+  target: https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/example-components.yaml
```

Comment:
This isn't using the shorter form used in the PR body, do we support that?

`target: https://github.com/spotify/backstage/plugins/catalog-backend/examples/example-components.yaml`

Reply:
Hm not sure that form actually exists. It was hand typed

Reply:
I know the different available forms have caused a lot of confusion internally. Hunch is that we should be pretty permissive in handling of the input, maybe supporting `raw` urls, but at least both https://github.com/spotify/backstage/blob/master/app-config.yaml and https://github.com/spotify/backstage/tree/master/app-config.yaml.

Then I'm wondering if we should be providing some shorthands, for example saying that we support the Golang import syntax, which would be just `"github.com/spotify/backstage/app-config.yaml"`.

Reply:
Oh nice, yeah absolutely, if you're at a computer maybe paste those possibilities in a ticket, otherwise I'll make one after the weekend

Reply:
Created #1279, we probably already support both `/tree/` and `/blob/` since we're not doing any special parsing of the URL afaik?

Reply:
Not sure, that will have to be verified. We convert them to the `raw` form and I am not sure the conversion works for both.

Reply:
Yeah we don't support /tree/, but we validate that it's /blob/ so I think it's fine. Adding support for /tree/ requires normalization before storage, or you'd end up with duplicates, so it's a bit more involved.